cms/templates/howitworks.html

```diff
@@ -146,7 +162,7 @@ <h2 class="sr">${_("Sign Up for {studio_name} Today!").format(studio_name=settin
 
     <ul class="list-actions">
       <li class="action-item">
-        <a href="${reverse('signup')}" class="action action-primary">${_("Sign Up &amp; Start Making an {platform_name} Course").format(platform_name=settings.PLATFORM_NAME)}</a>
+        <a href="${reverse('signup')}" class="action action-primary">${_("Sign Up & Start Making an {platform_name} Course").format(platform_name=settings.PLATFORM_NAME)}</a>
```

Comment:
@nedbat Do we have any way to properly deal with the article (i.e. a vs an) for variables?  It would seem you would want a variable or function for `an {platform_name}`.